docs/src/details/integration/renderers/eve.rst

```diff
@@ -0,0 +1,233 @@
+=========================
+EVE External GPU Renderer
+=========================
+
+EVE is a type of external GPU IC which accepts high-level drawing commands over SPI
+and outputs the rendered graphics to a display over parallel RGB.
+
+The advantage of using an EVE chip that the rendering responsibility is removed
+from the driving MCU so it can be a lower-spec part or dedicate more of its
+processing time to other tasks. The SPI interface is simpler to connect than
+parallel RGB especially if the EVE chip is integrated into a display assembly.
+
+LVGL features a renderer for EVE. LVGL UIs can be rendered by EVE
+and are effectively indistinguishable from the software renderer is most cases.
+
+See also the :ref:`ft81x` framebuffer driver. It drives the same EVE chips
+but is a simpler, more standalone implementation which uses software rendering
+and sends all the pixels over SPI so it is much slower.
+
+Limitations
+***********
+
+- Image format, size, and count limit.
+- Font format, size, and count limit.
+- The total number of tasks rendered per refresh has an upper limit.
+- Layers are not supported.
+
+Usage
+*****
+
+Board Parameters
+----------------
+
+Create a ``lv_draw_eve_target_parameters.h`` and add it to the include search
+path of your build.
+
+Here is an example ``lv_draw_eve_target_parameters.h`` for the Riverdi RVT50HQBNWC00-B
+which has a BT817Q --- the EVE chip.
+These parameters were taken from the collection of configs provided by the EVE
+library supporting this renderer.
+`See here <https://github.com/RudolphRiedel/FT800-FT813/blob/ee35fa45c37aed78dc4f1196b874ee1c8f80429f/EVE_config.h#L856-L872>`__.
+Check there for your board parameters.
+
+.. code-block:: c
+
+    #pragma once
+
+    #define EVE_HSIZE (800L)
```

Comment:
It's a little against how we usually handle configs. We should either use `lv_conf.h` for these, but as there are a lot of options passing them as a larger struct  to `lv_draw_eve_display_create()` might be better.